data-manipulation/checksum/luhn/luhn-credit-card-validation.yml

```diff
@@ -0,0 +1,37 @@
+rule:
+  meta:
+    name: luhn credit card validation
```

Comment:
we typically name rules to fit the pattern "this function may... [fill in here]", so i'd suggest the rule name "validate credit card number with luhn algorithm"

Reply:
Can do, I'll make the changes.  Thanks for the feedback.